less/index.less

```diff
@@ -53,7 +53,208 @@ table {
+@border-color-light: #dddddd;
+
+//font-size stack
+@text-small: 1em;
+@text-reg: 1.125em;
+@text-med: 1.25em;
+@text-large: 1.5em;
+@text-jumbo: 2em;
+
+
+// CSS Start
+
+
+// Global Element Selectors
+
+h1 {
+	color: #252122;
+	font-weight: 600;
+	font-size: @text-large;
+	letter-spacing: -0.035em;
+	line-height: 1;
+	margin: 0.5em 0;
 }
 
+h2, h3, h4, h5, h6 {
+	font-size: @text-med;
+	margin: 0.5em 0;
+}
+
+button {
+	background-color: #4CAF50; /* Green */
+	border: none;
+	border-radius: 0.2em;
+	color: white;
+	padding: 15px 32px;
+	text-align: center;
+	text-decoration: none;
+	display: inline-block;
+	font-size: @text-small;
+	margin: 12px 2px;
+	cursor: pointer;
+}
+
+p {
+	margin-bottom: 2em;
+	line-height: 1.4em;
+}
+
+img {
+	.max-width;
+    height: auto;
+}
+// Nested Container Selectors
+
+.container {
+	.max-width;
+	width: 70em;
+	margin: 0 auto;
+	// .red-border;
+
+		.nav {
+			margin: 0.5em 0 0 0;
+			
+
+			ul {
+				margin: 0;
+			}
+
+			li {
+				display: inline-block;
+				margin: 0 0.35em 0 0.35em;
+
+				@media (max-width:500px){
+					.display-flex;
+					flex-direction: column;
+					align-items: center;
+				}
+			}
+
+			a {
+				border-radius: 5px;
+					color: #5d5d5d;
+					text-decoration: none;
+					padding: 0.6em 1.2em 0.6em 1.2em;
+			}
+		} //end nav
+
+		.info-main {
+			.max-width;
+			background: #e8e9f1;
+			width: 67em;
+			margin: 0 auto;
+			margin-top: 1em;
+
+				section.intro {
+					margin-left: 1.5em;
+				}
+
+				section.intro h1 {
+					margin: 1em 0;
+				}
+
+				section.intro h4,p {
+					margin: .5em 0;
+				}
+		}
+
+		.row-info {
+			.display-flex;
+			margin-left: 1.5em;
+			justify-content: space-between;
+			margin-top: 1.3em;
+
+			.col-1 {
+				width: 45%;
+
+				table, th, td {
+					border: 1px solid grey;
+					border-collapse: collapse;
+					padding: 1em;
+				  }
+			}
+		} // end row
+
+		.work-history {
+			margin-left: 1.5em;
+
+			table {
+				font-family: arial, sans-serif;
+				border-collapse: collapse;
+				width: 98%;
+			  }
+			  
+			  td, th {
+				border: 1px solid @border-color-light;
+				text-align: left;
+				padding: 8px;
+			  }
+			  
+			  tr:nth-child(even) {
+				background-color: @border-color-light;
+			  }
+		} //end work history
+
+		.contact {
+			margin-left: 1.5em;
+
+			form {
+				max-width: 98%;
+				padding: 1em;
+				border: 1px solid@border-color-light;
+				border-radius: 1em;
+			  }
+			  
+			  form div + div {
+				margin-top: 1em;
+			  }
+			  
+			  label {
+				display: inline-block;
+				width: 90px;
+				text-align: left;
+			  }
+			  
+			  input, textarea {
+				font: 1em sans-serif;
+				width: 300px;
+				.max-width;
+				box-sizing: border-box;
+			  	border: 1px solid @border-color-light;
+			  }
+			  
+			  input:focus, textarea:focus {
+				border-color: @border-color-dark;
+			  }
+			  
+			  textarea {
+				vertical-align: top;
+			  	height: 5em;
+			  }
+
+		} // end contact
+
+} //end container
```

Comment:
good job following the specific instructions to nest everything inside the main container.
great idea to add the comments on the ending curly braces like you did too, keeps everything organized.